bounties_api/std_bounties/views.py

```diff
@@ -103,3 +103,21 @@ def get(self, request, address=''):
             'submissions_acceptance_rate': submissions_acceptance_rate,
         }
         return JsonResponse(profile_stats)
+
+
+class Tokens(APIView):
+    def get(self, request):
+        token_qs = {}
+        result = []
+        token_to_append = {}
+        token_count_agg = {}
+        token_count_agg = Bounty.objects.values('tokenSymbol','tokenContract',
```

Comment:
@murduk - if you print the value of token_count_agg, you'll notice a few things:

```
<QuerySet [{'tokenSymbol': 'ETH', 'tokenContract': '0x0000000000000000000000000000000000000000', 'tokenDecimals': 18, 'token': 2, 'count': 193}, {'tokenSymbol': 'WYV', 'tokenContract': '0x056017c55ae7ae32d12aef7c679df83a85ca75ff', 'tokenDecimals': 18, 'token': None, 'count': 2}]>
```
There are 2 tokens in that set.  One of those tokens does not have a token_id.  That is because the token table syncs with coinmarket cap and the token WYV doesn't get returned from the API.  As stated in the prompt:

`For example, we may have a token as part of a contract that does not exist yet on coinmarketcap.` - however, I see this can be confusing.

The ultimate result from the API does not include this 2nd WYV token, which we need.  Rather than look up via id, it's better to look it up via the `symbol` field.

Additionally, the `bounty in token_count_agg` may be misleading.  Typically, you would use the lowercase name of a model to represent an instance.  You'll see from documentation that values/aggregates in django do not actually return an instance of a bounty. For example, you cannot change a value and then call .save().

In this case, I would encourage better naming - probably `token_count` in `token_count_agg` is good.